scheduler/scheduler-examples/src/main/java/org/ow2/proactive/scheduler/examples/FTPConnector.java

```diff
@@ -158,7 +160,7 @@ public Serializable execute(TaskResult... results) throws IOException {
 
             }
         } catch (IOException e) {
-            throw new IOException("Connection issues. The hostname and/or ftpPort can be wrong.");
+            throw new IOException(e);
```

Comment:
You can remove this `catch (IOException e)` because it adds no information.

Reply:
I agree with justine.

Reply:
I agree with you :D